cfgov/jinja2/v1/_includes/molecules/related-metadata.html

```diff
@@ -135,13 +140,9 @@ <h3 class="h4">
    ========================================================================== #}
 
 {% macro _text(text) %}
-
-<p class="u-mb0">
     {{ parse_links(text.blob) | safe }}
-</p>
 {% endmacro %}
 
-{# TODO: This function is not used and can be removed. #}
```

Comment:
I think `page` is always available to templates. Is that what you mean? Some `page` data must be needed by the `related_metadata_tags` function, which must not be part of the template structure, since it's not imported here.